README.md

```diff
@@ -1009,3 +1011,30 @@ Here you can find references to modules and repositories contain examples of tes
 * ```minar``` module contains [test cases](https://github.com/ARMmbed/minar/tree/master/test) written without ```utest```. Note: ```utest``` may use ```minar``` for callback scheduling and can't be use to test ```minar``` itself.
 * ```mbed-drivers``` module contains [test cases](https://github.com/ARMmbed/mbed-drivers/tree/master/test) written with and without ```utest``` harness. Currently all ```mbed-drivers``` tests are using [build-in to ```htrun``` host tests](https://github.com/ARMmbed/htrun/tree/master/mbed_host_tests/host_tests).
 * And finally ```sockets``` module contains [test cases](https://github.com/ARMmbed/sockets/tree/master/test) with [custom host tests](https://github.com/ARMmbed/sockets/tree/master/test/host_tests).
+
+# Plugins
+
+In order to work with platforms for which the hardware is still under development, and hence may not have an mbed interface chip, some "hook" files are required.  Operation with these platforms is a matter for the platform development teams involved and is not, in general, supported by ARM.
+
+## SARA NBIOT EVK
+The SARA NBIOT EVK board must be connected to a Windows PC using a Segger JLink box, which is used for downloading code and resetting the board. The USB port on the EVK must also be connected to the same PC.  You will also require access to some proprietary tools that can be requested from u-blox.
+
+Once your mbed tests have been compiled, they may be executed with:
+
+`python RunTests.py`
```

Comment:
What is `RunTests.py`? Is this script necessary to run these tests with htrun? Is it basically a stand-in for greentea? If so, then it seems like this documentation shouldn't be in htrun. I would instead just include the details about the plugin itself, not about how to run a suite of tests (as this outside of the scope of htrun any way)